File: 119. Pascal's Triangle II/main.js
/**
 * @param {number} rowIndex
 * @return {number[]}
 */
var getRow = function (rowIndex) {
  let res = [1];

  for (let i = 1; i < rowIndex + 1; i++) {
    for (let j = 0; j < i - 1; j++) {
      res[j] += res[j + 1];
    }
    res.unshift(1);
  }

  return res;
};

File: 119. Pascal's Triangle II/main.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        res = [1]

        for i in range(1, rowIndex+1):
            for j in range(i-1):
                res[j] += res[j+1]
            res.insert(0, 1)
        
        return res